http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -576,7 +573,11 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
           case Some(resp) =>
             resp.jobReport.status match {
               case WsmJobStatus.Succeeded =>
-                deleteDiskAction
+                for {
+                  _ <- dbRef.inTransaction(clusterQuery.updateClusterStatus(runtime.id, RuntimeStatus.Deleted, ctx.now))
+                  _ <- logger.info(ctx.loggingCtx)("runtime is deleted successfully")
```

Comment:
While we're touching this maybe improve the log msg? :) 
```
"Runtime ${msg.runtimeId} is deleted successfully"
```

Reply:
good idea